### ISSUE TITLE
Visible name and accessible name do not match for the “Options” button in Windows.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#24304]
**Pre-requisite:**
Turn on Narrator: Ctrl + Win + Enter

**Test Environment:**
OS: Windows 11 Version 26H1 (OS Build 28000.1340)
Browser: Microsoft Edge Dev Version 146.0.3818.0 (64‑bit)
URL: https://www.highcharts.com/demo/stock/flags-general
Tools: Accessibility Insights for Windows
Screen Reader: Narrator

**User Impact:**
Screen reader users rely on accessible names to understand and identify controls. When the accessible name does not match the visible label, it causes confusion and makes it difficult for users to understand the purpose of the control, leading to a poor and inconsistent user experience for users with disabilities.

**Steps to Reproduce:**

1. Open the given URL in the Edge Dev browser and log in using valid credentials.
2. Navigate to the graph available on the page.
3. Turn on Narrator using Ctrl + Windows + Enter.
4. Press the Tab key repeatedly to move focus through the controls.
5. Move focus to the “Options” button associated with the graph.
6. Observe the screen reader narration for the focused control.

**Actual Result:**
The accessible name of the control is announced as “View chart menu”, while the visible label of the control is “Options”.

**Refer Attachment:**
1. Visible name and accessible name do not match for the “Options” button in Windows.png
2. Visible name and accessible name do not match for the “Options” button in Windows.mp4

![Image](https://github.com/user-attachments/assets/604f3e46-1b59-430d-b9db-7c1f4676ab19)
https://github.com/user-attachments/assets/be38ce3e-9063-42cb-b647-51d014cf5552

**Expected result:** 
The visible name and accessible name should be the same for the control, or at minimum, the accessible name should start with the visible name “Options”, to ensure consistency and clarity for assistive technology users.

**Comment by kamil-musialowski** [comment]
Hi there, thanks you for reporting that issue! Sounds like a straightforward fix. We will add this to our backlog to evaluate the ticket and set its priority :)
